lib/tasks/data.rake

```diff
@@ -0,0 +1,227 @@
+namespace :data do
+  desc "Creates departments and agencies"
+  task create_depts: :environment do
+    Department.all.each{ |d| d.home_node && d.home_node.delete }
+    Agency.all.each{ |a| a.home_node && a.home_node.delete }
+
+    Agency.delete_all
+    Department.delete_all
```

Comment:
This seems like it may be too destructive. What does this do to nodes in the section? What does it do to user roles? Do we need to keep them around?